### PR TITLE
README.md

### PR DESCRIPTION
**Pull request by smaftoul**
@kpolszewski here is it ! 
Sorry for the lag !
I would be happy if you try it and tell me if this ok for your usage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Docker distribution pruner
+
+This is a dockerfile to build the gitlab's docker distribution pruner ([https://gitlab.com/gitlab-org/docker-distribution-pruner]()).
+
+The image is published as an automated build on the docker hub, you can pull it with `docker pull wizbii/distribution-pruner`.
+
+## usage
+
+You can simply `docker run wizbii/distribution-pruner`, but the pruner needs the distribution config file.
+To have access the config file you can `docker run --volumes-from=registry wizbii/distribution-pruner -config /path/to/folder/of/you/registry/config.yml`
```
Comment (Baldinof) [inline README.md]: Maybe `--volumes-from=your-registry-container-id`?

Comment (smaftoul) [reply README.md]: `--volumes-from=your-registry-container` because it can be a name , right ?

Comment (Baldinof) [reply README.md]: 👌

**Comment by kpolszewski** [comment]
and that's what you call a good readme ;)
works great, thanks!

**Pull request merged**